pypeit/core/pydl.py

```diff
@@ -331,7 +329,7 @@ def from_dict(cls, bspline_dict):
         return cls(None,
                    nord=bspline_dict['nord'],
                    npoly=bspline_dict['npoly'],
-                   bkpt=np.array(bspline_dict['bkpt']),
+                   fullbkpt=np.array(bspline_dict['breakpoints']),
```

Comment:
This line is incorrect. fullbkpt is not a class attribute. It is breakpoint. Please look at the source code where self.breakpoints is set.